src/main/java/mate/academy/spring/service/UserMapper.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.spring.service;
+
+import mate.academy.spring.dto.UserResponseDto;
+import mate.academy.spring.model.User;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserMapper {
+    public static UserResponseDto parse(User user) {
```

Comment:
let's make this method non-static and autowire this dependency if we need it in the controller